packages/mds-policy-service/service/provider.ts

```diff
@@ -35,10 +35,13 @@ const serviceErrorWrapper = async <T>(method: string, exec: () => Promise<T>) =>
 
 export const PolicyServiceProvider: ServiceProvider<PolicyService, PolicyServiceRequestContext> & ProcessController = {
   start: async () => {
-    await Promise.all([PolicyRepository.initialize(), PolicyStreamKafka.initialize()])
+    await Promise.all([
+      PolicyRepository.initialize(),
+      ...(process.env.KAFKA_HOST ? [PolicyStreamKafka.initialize()] : [])
+    ])
```

Comment:
While the suggested change might be a bit more readable, it will not scale well if we need to add other conditional entries over time (for example, check out `writeTripMetadata` in mds-stream). I'd prefer to go with the approach we're using elsewhere, which is how I've implemented it here.

Reply:
@PeterSP I think it's readable as is. It's part of the established style.